Standards/scs-0004-v1-achieving-certification.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Regulations for achieving SCS-compatible certification
+type: Procedural
+status: Draft
+track: Global
+---
+
+## Introduction
+
+The Sovereign Cloud Stack (SCS) issues certificates with various scopes, among them _SCS-compatible IaaS_ (infrastructure as a service) and _SCS-compatible KaaS_ (Kubernetes as a service).
+
+This document details how a cloud service provider (henceforth also called operator) can attain such a certificate for one of their clouds.
+
+## Motivation
+
+As operator, I want to obtain a certificate with the scope SCS-compatible IaaS or SCS-compatible KaaS.
+
+## Regulations
+
+1. Each certificate issued pertains to a given cloud, a given scope, and a given version of that scope with a fixed expiry date. The certificate is only valid for that cloud and for the time frame that ends on that expiry date.
+
+2. The operator MUST include the official [SCS compliance test suite](https://github.com/SovereignCloudStack/standards/tree/main/Tests) (which does not require admin privileges) in their continuous test infrastructure (e.g., Zuul). The tests MUST be run at given intervals, depending on their resource-usage classification:
+
+    - _light_: at least nightly,
+    - _medium_: at least weekly,
+    - _heavy_: at least monthly.
+
+   For public clouds, it is recommended to offer the SCS project access to the infrastructure so the test suite runs can be triggered continuously by the SCS team.
+
+   Alternatively, and for non-public clouds, the results (log files) MUST be submitted to SCS by a mechanism of SCS' choice and need to be reproduced again on request by SCS.
```

Comment:
We should have a set of options for providing the results ready. On some airgapped infrastructures getting information like that out of it is nearly impossible as every information is classified. That may even lead to a letter with printed logs, that need to be destroyed in certain ways after reading.